### PR TITLE
CFY-3120 allow to configure hide groups via fabric env

### PR DESCRIPTION
**Pull request by nir0s**

### CODE REVIEW

**Comment by nir0s** [comment]
To the reviewer:

The name of the argument for hiding groups is currently `hide_output`. Not sure about the name. If there's a better suggestion, I'd be happy to hear it. It was `hide`, at first.. but it's too ambiguous.

File: fabric_plugin/tests/tests.py [hunk 1]
```diff
@@ -402,6 +426,66 @@ def test_failed_command(self):
...
+            task_mapping='fabric_plugin.tests.tests.module_task')
+
+    def test_hide_non_viable_groups_in_script(self):
+        try:
+            self._test_hide_in_settings(
+                execution_method='run_script',
+                script_path='scripts/script.py')
+            self.fail()
+        except TestException as ex:
+            self.assertIn("'hide_output': ('running', 'stdout')", str(ex))
```
Comment (nir0s) [inline fabric_plugin/tests/tests.py]: So.. `ex.message` could return `self.settings_merged` after which we could assert that the dict contains the relevant subset. Thing is.. we don't want to use `ex.message`. This is good enough...

File: fabric_plugin/tasks.py [hunk 2]
```diff
@@ -326,6 +336,18 @@ def get_script(download_resource_func, script_path):
...
 
 
+def _hide_output(groups):
+    possible_groups = (
+        'status', 'aborts', 'warnings', 'running', 'stdout', 'stderr', 'user'
+    )
+    groups = groups or ()
+    if any(group not in possible_groups for group in groups):
+        raise NonRecoverableError(
+            '`hide_output` must be a subset of {0} (Provided: {1})'.format(
```
Comment (nir0s) [inline fabric_plugin/tasks.py]: This was `Hide groups must be one of`.. which is simply not true and also doesn't indicate where the problem is originating from.

File: fabric_plugin/tests/tests.py [hunk 3]
```diff
@@ -402,6 +425,70 @@ def test_failed_command(self):
...
+
+    def test_hide_non_viable_groups_in_task(self):
+        self._test_hide_in_settings(
+            execution_method='run_task',
+            task_name='task')
+        self._test_hide_non_viable_groups(
+            execution_method='run_task',
+            task_name='task')
+
+    def test_hide_non_viable_groups_in_module(self):
```
Comment (dankilman) [inline fabric_plugin/tests/tests.py]: rename tests

**Pull request merged**